types/react-helmet-with-visor/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Whoa there, that's a lot of suppressions. While packages already in DT get a pass, new packages to DT generally have to conform to the lint rules. Could you make the `tslint.json` just be `{ "extends": "dtslint/dt.json" }` and fix any lint errors which may arise?

Reply:
Hi, unfortunately, running npm run lint react-helmet-with-visor locally results in this error (#24788 ), and non of the proposed solutions are working for me.